Sources/SotoSignerV4/HexEncoding.swift

```diff
@@ -108,3 +131,22 @@ extension UInt8 {
         }
     }
 }
+
+extension ContiguousBytes {
+    /// return a hexEncoded string buffer from an array of bytes
+    @_disfavoredOverload
+    @_spi(SotoInternal)
+    public func hexDigest() -> String {
+        self.withUnsafeBytes { ptr in
+            ptr.hexDigest()
+        }
+    }
+}
+
+extension Collection<UInt8> {
+    /// return a hexEncoded string buffer from an array of bytes
+    @_spi(SotoInternal)
+    public func hexDigest() -> String {
```

Comment:
```suggestion
    @inlinable
    public func hexDigest() -> String {
```

Reply:
You can't mark SPI function `@inlinable`. They are already public

Reply:
Oh sorry I take that back. Got mixed up with `@usableFromInline`